lib/token-autocomplete.ts

```diff
@@ -28,7 +28,8 @@ interface Options {
     minCharactersForSuggestion: number,
     allowCustomEntries: boolean,
     readonly: boolean,
-    optional: boolean
+    optional: boolean,
+    enableTabulator: boolean
```

Comment:
Mhhh finde das Setting bissl spezifisch, am Ende haben wir dann enableTabulator, enableEnter, enableArrows, ... wenns blöd läuft. Spontan fällt mir aber auch nichts wirklich mega gutes ein

Reply:
ja muss man schauen, evtl fällt uns in zukunft noch was anderes auf wo wir BE und FE unterscheiden müssen dann könnten wir da evtl. was zusammenfassen

Reply:
Oder halt sowas allgemeines wie `disabledShortcuts` oder `disabledKeyBindings`. Aber kp